client/src/partials/fiscal/fiscal.update.js

```diff
@@ -118,6 +119,7 @@ angular.module('bhima.controllers')
       session.saved = false;
       session.saveError = false;
       $scope.fiscal = angular.copy(editCache);
+      $route.reload();
```

Comment:
This sub controller can not have the responsibility of reloading the all page, let use a service to handle the Fiscal Year state.